next.config.js

```diff
@@ -6,7 +6,10 @@ module.exports ={
   },
   serverRuntimeConfig: {
     // Will only be available on the server side
-    API_ENDPOINT: 'https://app-development.plant-for-the-planet.org'
+    API_ENDPOINT: 'https://app-development.plant-for-the-planet.org',
+    OAUTH0_DOMAIN: "",
```

Comment:
`AUTH0_CUSTOM_DOMAIN`

Reply:
This, and the other reviews should be resolved as part of the latest commit.